src/fetch/fetch-requests.js

```diff
@@ -1,4 +1,9 @@
 export const fetchCards = () => {
     return fetch('https://empress-tarot-api.herokuapp.com/cards')
     .then(res => res.json())
+}
+
+export const fetchCard = (id) => {
+    return fetch(`https://empress-tarot-api.herokuapp.com/cards/${id}`)
+    .then(res => res.json())
```

Comment:
we might be able to get this info directly from the 'deck' in app without a separate fetch request